people/mahnoor_fatima.txt

```diff
@@ -0,0 +1 @@
+This is a random sentence - I guess!
```

Comment:
This is a good sentence, but shouldn't it be located in `part4-changing_the_code/people/` and not in the root directory of the repository?